### PR TITLE
Add use_bias argument for BPR, WBPR, SBPR models

### PR DESCRIPTION
**Pull request by lthoang**
### Description

Add use_bias argument for BPR, WBPR, SBPR models.

This will allow user to specify whether to use bias terms in their own experiments.

### Checklist:

- [ ] I have added tests.
- [x] I have updated the documentation accordingly.
- [ ] I have updated `README.md` (if you are adding a new model).
- [ ] I have updated `examples/README.md` (if you are adding a new example).
- [ ] I have updated `datasets/README.md` (if you are adding a new dataset).

### CODE REVIEW

File: cornac/models/bpr/recom_bpr.pyx [hunk 1]
```diff
@@ -279,10 +288,10 @@ class BPR(Recommender):
 
         """
         if item_idx is None:
-            known_item_scores = np.copy(self.i_biases)
+            known_item_scores = np.copy(self.i_biases) if self.use_bias else np.zeros_like(self.i_biases)
             fast_dot(self.u_factors[user_idx], self.i_factors, known_item_scores)
             return known_item_scores
         else:
-            item_score = self.i_biases[item_idx]
+            item_score = self.i_biases[item_idx] if self.use_bias else 0.
```
Comment (saghiles) [inline cornac/models/bpr/recom_bpr.pyx:295]: Same comment as above.

**Comment by tqtg** [comment]
LGTM. Just needs to double-check the examples to make sure the algorithms run properly.

**Review by tqtg — APPROVED** [review]

**Comment by saghiles** [comment]
This PR looks good to me.

**Comment by tqtg** [comment]
@lthoang please feel free to merge if no more changes

**Pull request merged**